config/security/admin_role_hierarchy.yaml

```diff
@@ -63,4 +67,12 @@ security:
         PERMISSION_SFS_USER_ADMIN_INVITATIONS_DETAILS:
             - ROLE_ADMIN_INVITATIONS_DETAILS
         PERMISSION_SFS_USER_ADMIN_HISTORY_LIST:
-            - ROLE_ADMIN_ACCESS_HISTORY_LIST
\ No newline at end of file
+            - ROLE_ADMIN_ACCESS_HISTORY_LIST
+        PERMISSION_SFS_USER_ADMIN_USERS_CONFIRM:
```

Comment:
This is to support deprecated roles, those ones are new, so we don't need to add them as old format